Libs/Core/ctkUtils.cpp

```diff
@@ -504,3 +504,18 @@ QTextStream& ctk::endl(QTextStream &stream)
   return stream << QLatin1Char('\n') << ctk::flush;
 #endif
 }
+
+//------------------------------------------------------------------------------
+QModelIndex ctk::modelChildIndex(QAbstractItemModel* item, const QModelIndex &parent, int row, int colum)
+{
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 8, 0))
+  if (!item)
+    {
+    return QModelIndex();
+    }
+  return item->index(row, colum, parent);
```

Comment:
See https://doc.qt.io/qt-5/qmodelindex-obsolete.html#child

Reply:
This commit was moved into a dedicated PR for further review. See https://github.com/commontk/CTK/pull/1011